jdaviz/configs/specviz2d/helper.py

```diff
@@ -125,8 +128,14 @@ def load_data(self, spectrum_2d=None, spectrum_1d=None, spectrum_1d_label=None,
         if spectrum_2d is not None:
             self.app.load_data(spectrum_2d, parser_reference="mosviz-spec2d-parser",
                                data_labels=spectrum_2d_label,
-                               show_in_viewer=show_in_viewer,
+                               show_in_viewer=False,
                                add_to_table=False)
+
+            # Passing show_in_viewer into app.load_data does not work anymore,
+            # so we force it to show here.
+            if show_in_viewer:
```

Comment:
Honestly, I don't know and I don't feel like jumping into the rabbit hole if this PR is good enough. If you really want this investigated, let's open a follow-up ticket and story point it?

Reply:
I agree.  This is good enough.  I'd rather leave it and push more on Specviz2d getting subsumed by a Mosviz refactor.

Reply:
I agree that we should probably follow this up (and perhaps consider a true refactor as tech debt), as ignoring it will just make these more difficult to handle in the future.  But that doesn't need to hold up the immediate bugfix so we can get this case working again.

Reply:
I created a follow-up issue at #1320 . FYI.